driver/driver_test.go

```diff
@@ -330,7 +330,7 @@ func TestStmt_QueryTooManyParams(t *testing.T) {
 		values[i] = int64(1)
 	}
 	_, err = stmt.Query(values)
-	require.Errorf(t, err, "too many parameters (256) max = 255")
+	require.NoError(t, err)
```

Comment:
Same here. It would be good to do something like:

- insert 300 rows, with values ranging from 1 to 300
- run the SELECT statement in the test
- check that exactly 256 rows are returned, with the expected values

Perhaps it'd be worth using a slightly larger value than 256, say 260 or 270, just to test not only the boundary value but also beyond.